### ISSUE TITLE
Explore learning rate on gradient descent.

### ISSUE THREAD

**Issue by Heitor-Carvalho** [Heitor-Carvalho/ML#3]
-  Check if they converge to the close form solution
